docs/src/language_servers.md

```diff
@@ -290,8 +294,16 @@ Follow installation instructions on [LSP-json](https://github.com/sublimelsp/LSP
 
 ## LaTeX
 
+There are multiple options:
+
+### TexLab
+
 Follow installation instructions on [LSP-TexLab](https://github.com/sublimelsp/LSP-TexLab).
 
+### LTeX
```

Comment:
"LTeX" is not a language, no? We seem to be ordering these sections on language, not language server?

Reply:
"LTeX" is the name of the language server.

When there is "There are multiple options" I put the name of the language server for each option.

Reply:
But the other sections are (mainly) languages. The exception seems to be ESLint. LTeX-LS serves latex and markdown.

Reply:
It's still under the LaTeX language section. This heading with the language server name is one level smaller, similar to the language server sections under Python or Ruby for example.